src/components/shared/cellRenderers/connectionStatusRenderer/connectionStatusRenderer.js

```diff
@@ -7,15 +7,14 @@ import { svgs } from 'utilities';
 
 import '../cellRenderer.css'
```

Comment:
So far the css for cell rendering is going into a single scss file.  Therefore, the extra layer of having folder for each renderer is not necessary.

Reply:
It's a good point. We should probably move the shared form controls to obey the folder structure. I don't l like folders with a single file in them so perhaps the pattern should be, if a single file, just have that file, it there are property files (like css), bunch them in a folder.